src/Application/Service/CatalogDumpService.php

```diff
@@ -0,0 +1,123 @@
+<?php
+
+declare(strict_types=1);
+
+namespace LinkedDataSets\Application\Service;
+
+use EasyRdf\Graph;
+use EasyRdf\RdfNamespace;
+use EasyRdf\Resource;
+use Laminas\Log\Logger;
+use Laminas\Log\LoggerInterface;
+use Laminas\ServiceManager\Exception\ServiceNotFoundException;
+
+final class CatalogDumpService
+{
+    // in php 8.1 convert to enum
+    const DUMP_FORMATS = [
+        "turtle" => "ttl",
+        "ntriples" => "nt",
+        "jsonld" => "jsonld",
+        "rdfxml" => "xml",
+    ];
+
+    protected ?Logger $logger = null;
+    protected $serverUrl;
+    protected $id;
+    public function __construct(
+        LoggerInterface $logger,
+        $viewHelperManager
+    )
+    {
+        $this->logger = $logger;
+        $this->serverUrl = $viewHelperManager->get('ServerUrl');
+        if (!$this->serverUrl) {
+            throw new ServiceNotFoundException('The serverUrl service is not found');
+        }
+    }
+
+    public function dumpCatalog(int $id): void
+    {
+        $this->id = $id;
+        $apiUrl = $this->serverUrl->getScheme() . '://' . $this->serverUrl->getHost() . "/api/items/{$this->id}";
+
+        # Step 0 - create graph and define prefix schema:
+        RdfNamespace::set('schema', 'https://schema.org/');
+        $graph = new Graph(); //dep injection?
+
+        # Step 1 - get data catalog
+            $graph->parse($apiUrl, 'jsonld');
+
+        foreach ($graph->resources() as $resource) {
+            # Step 2 - get all datasets which are part of the data catalog
+            $datasets = $resource->allResources("schema:dataset");
+            foreach ($datasets as $dataset) {
+                $dataset_uri = $dataset->getUri();
+                $graph->parse($dataset_uri, 'jsonld');
+            }
+
+            # Step 3 - get all distribution which are part of datasets
+            $distributions = $resource->allResources("schema:distribution");
+
+            foreach ($distributions as $distribution) {
+                $distribution_uri = $distribution->getUri();
+                $graph->parse($distribution_uri, 'jsonld');
+            }
+
+            # Step 4 - get all publishers and creators (of data catalog and datasets)
+            $publishers = $resource->allResources("schema:publisher");
+
+            foreach ($publishers as $publisher) {
+                $publisher_uri = $publisher->getUri();
+                $graph->parse($publisher_uri, 'jsonld');
+            }
+
+            $creators = $resource->allResources("schema:creator");
+
+            foreach ($creators as $creator) {
+                $creator_uri = $creator->getUri();
+                $graph->parse($creator_uri, 'jsonld');
+            }
+        }
+
+        # Step 5 - remove Omeka classes and properties (o:)
+        $this->removeOmekaTags($graph);
+
+        # Step 6 - output the graph in several serializations
+        $this->dumpSerialisedFiles($graph);
+    }
+
+    // Maybe this step isn't needed, Bob said on 11th April.
```

Comment:
I'd refrain from adding this kind of info to the actual code base and offload it to the project board. This comment has meaning to you, and a little bit to me, but any contributor outside our organization would be clueless as to what this means.